src/ng-select/lib/ng-select.component.ts

```diff
@@ -515,7 +522,17 @@ export class NgSelectComponent implements OnDestroy, OnChanges, AfterViewInit, C
         return empty && this._isTypeahead && !this.searchTerm && !this.loading;
     }
 
+    handleSearchComposition(isCompositionStart: boolean, term: string) {
```

Comment:
split this method to separate methods, eg:. `onCompositionStart` and `onCompositionEnd`

Reply:
Fixed in ab098331563ac33f6e50204f9088a491c7742493